apps/tlon-mobile/index.js

```diff
@@ -3,6 +3,7 @@ import { addCustomEnabledLoggers } from '@tloncorp/shared';
 import { setStorage } from '@tloncorp/ui';
 import { registerRootComponent } from 'expo';
 import 'expo-dev-client';
+import 'react-native-get-random-values';
```

Comment:
This ended up being extraneous, right?

Reply:
This is necessary for our s3 methods to work.